packages/yoastseo/src/languageProcessing/researches/getProductSKUData.js

```diff
@@ -15,5 +16,8 @@ export default function( paper ) {
 	if ( customData.hasOwnProperty( "doAllVariantsHaveSKU" ) ) {
 		productData.doAllVariantsHaveSKU = customData.doAllVariantsHaveSKU;
 	}
+	if ( customData.hasOwnProperty( "variantIdentifierDataIsValid" ) ) {
```

Comment:
Shouldn't these three instances of the property for valid identifier data be identical, as in the case of `doAllVariantsHaveSKU`?

Reply:
It works this way because I'm referring to the same names that were given to the variables on the WooCommerce side. It's also this way in `getProductIdentifierData.js`. I agree that keeping the naming identical would be nicer, but if we change the naming we would also need to retest the grey bullet condition for the Product identifiers assessment in this PR. So maybe the extra testing is not worth a small change in naming?